lib/ansible/modules/files/find.py

```diff
@@ -425,8 +429,8 @@ def main():
 
                     try:
                         st = os.lstat(fsname)
-                    except Exception:
-                        msg += "%s was skipped as it does not seem to be a valid file or it cannot be accessed\n" % fsname
+                    except (IOError, OSError) as e:
+                        msg += "Skipped entry '%s' due to this access issue: %s\n" % (fsname, to_text(e))
```

Comment:
win_find adds these types of warnings to the warning key on the module, maybe we should do the same here.

Reply:
might be nicer change, i was just trying to clarify the message .. unsure if that would be backward breaking, as people might be inspecting results.

Reply:
at this point, im keeping existing msg, since i've found a couple of examples that look for it, but im also adding warnings at least to the 'paths' part.